buildAndReleaseTask/tests/_suite.ts

```diff
@@ -18,7 +18,7 @@ describe("Pulumi task tests", () => {
         assert.equal(tr.errorIssues.length, 0, "should have no errors");
 
         console.log("***Printing test output***");
-        console.log(tr.stdout);
+        //console.log(tr.stdout);
```

Comment:
Since this is not needed, let's remove this commented-out line.

Reply:
Will remove the commented out sections.